frontend/src/components/eMIB/Emib.jsx

```diff
@@ -28,6 +28,7 @@ let STRINGS = new LocalizedStrings({
     howToPageTitle: "HowTo Page",
     emailInstructions: "Email Instuctions",
     taskInstructions: "Task Instuctions",
+    howToNextButton: "Start timer and test",
```

Comment:
I find the length of this button a little bit long, especially in french. Maybe we could just have a button 'Start Test' and add a note saying that there will be a timer during the test?

Reply:
discussed offline; decided to crop button to just say "Start Test". Other PSC tests only say "Start Test"